src/lib.rs

```diff
@@ -6,6 +6,7 @@ mod lexer;
 mod module;
 mod parser;
 mod scope;
+mod test_utils;
```

Comment:
Since this is only for tests, it might make sense to mark this with the `#[cfg(test]` attribute macro. That way it isn't possible to accidentally leak these values into non-test code.